src/main/java/io/airbrake/javabrake/NoticeBuilder.java

```diff
@@ -0,0 +1,74 @@
+package io.airbrake.javabrake;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class NoticeBuilder {
+  Map<String, Object> context;
+  Map<String, Object> params;
+  Map<String, Object> session;
+  Map<String, Object> environment;
+  Throwable exception;
```

Comment:
The `Throwable exception` can be `final` since it cannot be updated and is always required.

Reply:
https://github.com/airbrake/javabrake/blob/7541aff0b22cd5cab9cb185fec9ba2eacfc48f19/src/main/java/io/airbrake/javabrake/Notice.java#L64
Here we are updating the exception object.

Reply:
👍